### PR TITLE
add yamllint format

### PR DESCRIPTION
**Pull request by colinrymer**

### CODE REVIEW

File: fmts/yamllint.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+func init() {
+	const lang = "yaml"
+
+	register(&Fmt{
+		Name: "yamllint",
+		Errorformat: []string{
+			`%f:%l:%c: %m`,
+		},
+		Description: "linter for YAML files",
```
Comment (mgrachev) [inline fmts/yamllint.go:11]:
```suggestion
		Description: "(yamllint -f parsable) A linter for YAML files",
```

**Review by mgrachev — COMMENTED** [review]
@colinrymer 👋 Thank you for your contribution! ✌️

I have left a comment. Please look at it 👀

**Review by mgrachev — APPROVED** [review]

**Comment by mgrachev** [comment]
@colinrymer Thank you! 👍

**Comment by review-dog** [comment]
Hi, @colinrymer! We merged your PR to reviewdog! 🐶
Thank you for your contribution! ✨

**We just invited you to join the @reviewdog organization on GitHub.**
Accept the invite by visiting https://github.com/orgs/reviewdog/invitation.
By joining the team, you'll be a part of reviewdog community and can help the maintainance of reviewdog.

Thanks again!

**Pull request merged**